server/ladders-local.ts

```diff
@@ -15,59 +15,57 @@
 
 'use strict';
 
-/** @type {typeof import('../lib/fs').FS} */
-const FS = require(/** @type {any} */('../.lib-dist/fs')).FS;
+import {FS} from '../lib/fs';
 
 // ladderCaches = {formatid: ladder OR Promise(ladder)}
 // Use Ladders(formatid).ladder to guarantee a Promise(ladder).
 // ladder is basically a 2D array representing the corresponding ladder.tsv
 //   with userid in front
-/** @typedef {[string, number, string, number, number, number, string]} LadderRow [userid, elo, username, w, l, t, lastUpdate] */
-/** @type {Map<string, LadderRow[] | Promise<LadderRow[]>>} formatid: ladder */
-let ladderCaches = new Map();
+type LadderRow = /** [userid, elo, username, w, l, t, lastUpdate */
+[string, number, string, number, number, number, string];
+type Cache = Map<string, LadderRow[] | Promise<LadderRow[]>>;
```

Comment:
```suggestion
```

Reply:
We use the type `Cache` twice, isn't it better to just make it a type since it's a bit verbose?

Reply:
Wait, we do? I didn't know. That's fine, then.

Reply:
Just don't call it `Cache`. Namespacing is important. Call it `LadderCache`.

Reply:
Oh, wait, we don't actually use it twice. We only do because you wrote the code wrong.

Reply:
Sure.

Reply:
I didn't see the last comment lol, @Zarel, with your review shouldn't we still give it a type though?

Reply:
No? Like I mentioned in other places, you shouldn't be using types where they can be inferred.